<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Help</class>
 <widget class="QWidget" name="Help">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>160</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="main_btn">
      <property name="text">
       <string>主页</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="mail_btn">
      <property name="text">
       <string>关于收发邮件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="safe_btn">
      <property name="text">
       <string>关于安全</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="design_btn">
      <property name="text">
       <string>关于界面</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>551</width>
     <height>441</height>
    </rect>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
